mod-source-record-manager-server/src/main/java/org/folio/services/afterprocessing/AdditionalFieldsUtil.java

```diff
@@ -292,6 +292,17 @@ private static MarcReader buildMarcReader(Record record) {
     return new MarcJsonReader(new ByteArrayInputStream(record.getParsedRecord().getContent().toString().getBytes(StandardCharsets.UTF_8)));
   }
 
+  public static boolean hasIndicator(Record record, char subfield) {
+    MarcReader reader = buildMarcReader(record);
+    if (reader.hasNext()) {
+      org.marc4j.marc.Record marcRecord = reader.next();
+      VariableField variableField = getSingleFieldByIndicators(marcRecord.getVariableFields(TAG_999), INDICATOR, INDICATOR);
+      return variableField != null
```

Comment:
I'm recomending use Objects.nonNull

Reply:
obj != null and Objects.nonNull are the same. I used this wrapper method to make it more clear, thank you @Abdulkhakimov !